chainer/functions/loss/mean_absolute_error.py

```diff
@@ -53,5 +58,36 @@ def mean_absolute_error(x0, x1):
             A variable holding an array representing the mean absolute
             error of two inputs.
 
+    Examples:
```

Comment:
Could you use `.. admonition:: Example`?  See e.g. `chainer/functions/activation/relu.py`